lib/identity/client.js

```diff
@@ -212,6 +230,18 @@ class Client extends PartialClient {
     const newSecretCreated = this.createSecret(newSecret)
     return newSecretCreated
   }
+  /**
+   * downloadFile
+   *
+   * @param {Object} recordId
+   *
+   * @return {string}
+   */
+  async downloadFile(recordId) {
```

Comment:
yeah, instead of this, probably return the platform agnostic `File()` object and in documentation demonstrate the helpers for turning that into items for use for the demonstrated platform.